ldms/src/sampler/zfs_topvdevs/Makefile.am

```diff
@@ -0,0 +1,21 @@
+pkglib_LTLIBRARIES =
+lib_LTLIBRARIES =
+dist_man7_MANS =
+dist_man1_MANS =
+
+AUTOMAKE_OPTIONS = subdir-objects
+
+AM_CPPFLAGS = @OVIS_INCLUDE_ABS@ \
+	      -I/usr/include/libspl \
+	      -I/usr/include/libzfs
+
+AM_LDFLAGS = @OVIS_LIB_ABS@
+COMMON_LIBADD = -lsampler_base -lldms -lovis_util -lcoll \
+		@LDFLAGS_GETTIME@
+
+if ENABLE_PROCNETDEV
+libzfs_topvdevs_la_SOURCES = zfs_topvdevs.c ../zfs_common/zfs_common.c
+
+libzfs_topvdevs_la_LIBADD = $(COMMON_LIBADD)
+pkglib_LTLIBRARIES += libzfs_topvdevs.la
+endif
```

Comment:
You also need to include the man page:

dist_man7_MANS = Plugin_zfs_topvdevs.man